bin/runtime/src/lib.rs

```diff
@@ -79,7 +79,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("aleph-node"),
     impl_name: create_runtime_str!("aleph-node"),
     authoring_version: 1,
-    spec_version: 73,
+    spec_version: 14_000_000,
```

Comment:
If we ever reach `>0` major version, then I think we might get into trouble with running beyond u32